### DISCUSSION TITLE
Custom canonical tag

### DISCUSSION THREAD

**Discussion by angelinekwan** [pradyunsg/furo#656] [Potential Issue] [answered]
Due to the [Issue 9730](https://github.com/sphinx-doc/sphinx/issues/9730) where upstream Sphinx doesn't construct canonical links correctly for the DirectoryHTMLBuilder, make it possible for furo theme to overwrite the canonical url. 

Similar to other theme has been making a fix on the theme level.
- [Sphinx RTD](https://github.com/readthedocs/sphinx_rtd_theme/blob/6beecce44759257ed584f2fdb0357f2aec0ad01a/sphinx_rtd_theme/layout.html#L62-L77)
- [Sphinx awesome](https://github.com/kai687/sphinxawesome-theme/pull/1262)

**Comment by pradyunsg** [comment]
_sigh_

Yea, let's do this.

**Reply by pradyunsg** [reply, marked as answer]
#682 for tracking this.
